lib/tools/test/tprof_SUITE.erl

```diff
@@ -396,23 +435,26 @@ hierarchy() ->
     [{doc, "Tests tracing for process hierarchy"}].
 
 hierarchy(Config) when is_list(Config) ->
-    {ok, _Srv} = tprof:start_link(#{type => call_memory}),
-    Traced = tprof:enable_trace({all_children, kernel_sup}),
+    {ok, Srv} = start_link(Config, #{type => call_memory}),
+    Traced = enable_trace(Config, Srv, {all_children, kernel_sup}, #{ set_on_spawn => false }),
     ?assert(Traced > 5),
-    ?assert(tprof:set_pattern(code_server, '_', '_') > 5),
+    ?assert(set_pattern(Config, Srv, code_server, '_', '_') > 5),
     _ = code:get_path(), %% makes a call to code_server
     %% disabling all processes tracing should return more than "children of"
-    ?assert(tprof:disable_trace(processes) > Traced),
-    {call_memory, Profile} = tprof:collect(),
-    tprof:stop(),
+    Disabled = disable_trace(Config, Srv, all),
+    ct:log("~p",[{Disabled, Traced}]),
```

Comment:
Minor nit, this just looks like a debug printout leftover.

Reply:
Yes, I'll remove them once all tests pass.

Reply:
I still see some, unless you wanted to keep it for future debugging.

Reply:
I've left it for now to debug tests... I'm having issues with timeouts on our 200+ core machines.